actiontext/lib/action_text/fixture_set.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module ActionText
+  class FixtureSet
+    def self.attachment(fixture_set_name, label, column_type: :integer)
+      signed_global_id = ActiveRecord::FixtureSet.signed_global_id(fixture_set_name, label, column_type: column_type, for: ActionText::Attachable::LOCATOR_NAME)
```

Comment:
Maybe this is a step too far and https://github.com/rails/rails/pull/40289/commits/7dfd9773b95351655ed44b5ea2648ddf2e58c9d2 is better, but I've pushed up the `signed_global_id` part to `ActiveRecord::FixtureSet`.

The tests for ActionText still pass, though I wasn't sure on where or how to add an ActiveRecord-side test for the new method.